nni/tools/nnictl/nnictl.py

```diff
@@ -167,6 +167,12 @@ def parse_args():
     parser_load_experiment.add_argument('--searchSpacePath', '-s', required=False, help='the path of search space file for \
                                    loaded experiment, this path contains file name. Default in $codeDir/search_space.json')
     parser_load_experiment.set_defaults(func=load_experiment)
+    #view an NNI experiment
```

Comment:
I'm a little confused. Where is the old `view`? As far as I remembered, there has already been a `view`.

Reply:
Seems they are `nnictl experiment view` and `nnictl view`.

Reply:
That's really confusing. Why can't we merge these two?

Reply:
unity command into `nnictl view`